### PR TITLE
allow for insertion of dummy entry points into the local table

### PR DESCRIPTION
**Pull request by aeyakovenko**
* Needed for #341.  Create a dummy entry with public key `0...`, but with a valid gossip address that we can ask for updates.  This will allow validators to discover the full network by just knowing a single node's gossip address without knowing anything else about their identity. 
* once we start removing dead validators this entry should get purged since we will never see a message from public key 0, #344

### CODE REVIEW

File: src/crdt.rs [hunk 1]
```diff
@@ -134,6 +133,17 @@ impl ReplicatedData {
             repair_addr,
         )
     }
+    pub fn new_entry_point(gossip_addr: SocketAddr) -> Self {
```
Comment (garious) [inline src/crdt.rs]: What do you mean by `entry_point` here? Would `new_from_base_addr` make more sense?

Comment (aeyakovenko) [reply src/crdt.rs]: By entry point I meant that it’s a gossip network entry point, it’s a dummy entry that had an invalid public key.  It’s used to attach to the network initially and will eventually get purged

Comment (garious) [reply src/crdt.rs]: Can you add code comments? You don't add a test in this PR and the only explanation is in the PR description and the review comment above.

Comment (garious) [reply src/crdt.rs]: Nevermind, I just didn't grok the test.

**Pull request merged**